tests/v2/validation/resource/terraform/v1charts/target/main.tf

```diff
@@ -0,0 +1,149 @@
+#Target cluster for v1 charts installation
+terraform {
+  required_version = ">= 0.14"
+  required_providers {
+    rancher2 = {
+      source = "rancher/rancher2"
+    }
+  }
+}
+
+provider "rancher2" {
+  api_url   = var.rancher_api_url
+  token_key = var.rancher_token_key
+}
+
+resource "rancher2_cluster_sync" "v1target-sync" {
+  cluster_id =  var.cluster_id
+}
+
+resource "local_file" "sync_kube_config" {
+    depends_on = [rancher2_cluster_sync.v1target-sync]
+    content  = rancher2_cluster_sync.v1target-sync.kube_config
+    filename = "kube_config_provisioned_cluster.yaml"
+}
+
+
+resource "rancher2_namespace" "longhorn-system-namespace" {
+  depends_on = [rancher2_cluster_sync.v1target-sync,local_file.sync_kube_config]
```

Comment:
Formatting nitpick: can we put each dependency on a separate line, or at least space out each one so they  can be easily identified as separate items?

Reply:
The depends_on section for each resource is now located at the bottom of each resource block with each dependencies on its own line
	- As per the example here: https://www.terraform.io/language/meta-arguments/depends_on